ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -375,6 +446,52 @@ private static Format parseFormat(final String formatString) {
         }
       }
 
+      private static int parseNumThreads(final String numThreadsString) {
+        try {
+          final int result = Integer.valueOf(numThreadsString, 10);
+          if (result < 0) {
```

Comment:
```suggestion
          if (result <= 0) {
```
And similarly for message rate below.